bears/python/BanditBear.py

```diff
@@ -24,18 +26,24 @@ class BanditBear:
     def create_arguments(filename, file, config_file,
                          bandit_skipped_tests: typed_list(str)=
                          ('B105', 'B106', 'B107', 'B404', 'B603', 'B606',
-                          'B607')):
+                          'B607'),
+                         bandit_selected_tests: typed_list(str)=()):
         """
-        :param bandit_skipped_tests:
-            The IDs of the tests ``bandit`` shall not perform. You can get
-            information about the available builtin codes at
-            https://github.com/openstack/bandit#usage.
+        :param bandit_skipped_tests: The IDs of the tests ``bandit`` shall
+                                     not perform.
+        :param bandit_selected_tests:  The IDs of the tests ``bandit`` shall
```

Comment:
A redundant space here

Reply:
Thank you @li-boxuan. Any other review? 😛